CMakeLists.txt

```diff
@@ -67,9 +67,11 @@ IF (BUILD_FUZZER)
     ENDIF()
     ADD_EXECUTABLE(quicly-fuzzer-packet fuzz/packet.cc ${PICOTLS_OPENSSL_FILES})
     SET(LIB_FUZZER "${CMAKE_CURRENT_BINARY_DIR}/libFuzzer.a")
-    SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fno-omit-frame-pointer -fsanitize=address,undefined -fsanitize-coverage=edge,indirect-calls,8bit-counters")
-    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fno-omit-frame-pointer -fsanitize=address,undefined -fsanitize-coverage=edge,indirect-calls,8bit-counters")
-    SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_C_FLAGS}")
+    SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fno-omit-frame-pointer -fsanitize=address,undefined")
+
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fno-omit-frame-pointer -fsanitize=address,undefined")
+
+    SET(CMAKE_EXE_LINKER_FLAGS "-v ${CMAKE_C_FLAGS}")
```

Comment:
What is `-v`?

Reply:
-v was just for my testing. I've removed that now.